### DISCUSSION TITLE
Self-nomination to Maintainer role

### DISCUSSION THREAD

**Discussion by ryanmoran** [paketo-buildpacks/feedback#124] [Node.js Team]
Hey @paketo-buildpacks/nodejs-maintainers! I'd like to propose adding myself as a maintainer for the Node.js buildpacks. I've contributed significantly to these buildpacks over time and believe that I can be a helpful addition to the team. Here are my contribution stats:

https://github.com/paketo-buildpacks/node-engine/graphs/contributors
https://github.com/paketo-buildpacks/node-run-script/graphs/contributors
https://github.com/paketo-buildpacks/node-start/graphs/contributors
https://github.com/paketo-buildpacks/npm-install/graphs/contributors
https://github.com/paketo-buildpacks/npm-start/graphs/contributors
https://github.com/paketo-buildpacks/yarn/graphs/contributors
https://github.com/paketo-buildpacks/yarn-install/graphs/contributors
https://github.com/paketo-buildpacks/yarn-start/graphs/contributors

On each of these repos, I'm consistently in the top 3 non-bot contributors.

Can I get a 👍 / 👎 vote?

**Comment by arjun024** [comment]
Congrats @ryanmoran. With unanimous vote of maintainers, you're now a Node.js subteam maintainer!
cc @paketo-buildpacks/steering-committee.
